lib/active_merchant/billing/gateways/nuvei.rb

```diff
@@ -315,7 +338,7 @@ def parse(body)
       end
 
       def success_from(response)
-        response[:status] == 'SUCCESS' && response[:transactionStatus] == 'APPROVED'
+        response[:status] == 'SUCCESS' && (response[:transactionStatus] == 'APPROVED' || response[:transactionStatus] == 'PENDING')
```

Comment:
❓ Question:

I'm seeing a `notification_url` field + a 'pending' status on the transaction, does that mean that ACH payments on this gateway are async?